web_easy_switch_company/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+web_easy_switch_company
+=======================
+
+This module extend web backend and allow user to switch to another company more easyly.
+
+Note: Due to licence incompatibility, you have not the legal right to install this module on enterprise version ...
```

Comment:
According to https://github.com/odoo/odoo/issues/9973#issuecomment-163199596 this sentense is not necessary.